packages/integration-tests/projects/suite-web/tests/suite/bug-report-form.test.ts

```diff
@@ -0,0 +1,40 @@
+// eslint-disable @typescript-eslint/naming-convention
+// @group:suite
+// @retry=2
+
+describe('Stories of bug report forms', () => {
+    beforeEach(() => {
+        cy.task('startEmu', { wipe: true, version: '2.4.3' });
```

Comment:
unless there is a reason, test should not have pinned firmware version

Reply:
Thank you, that was not intentional and overlooked!